src/main/java/edu/cornell/kfs/concur/dto/ConcurStandardAccountingExtractDTO.java

```diff
@@ -0,0 +1,6 @@
+package edu.cornell.kfs.concur.dto;
+
+public class ConcurStandardAccountingExtractDTO {
```

Comment:
Had some back and forth conversation with Jay on this, neither of us feels strongly about going one way or another. There are plain POJOs exist in the businessobject package, but it also makes sense to have this in a dto package since it is not persisted.